src/edit_view.rs

```diff
@@ -109,153 +195,136 @@ impl EditView {
         file_name: Option<String>,
         view_id: &str,
     ) -> Rc<RefCell<Self>> {
-        let da = DrawingArea::new();
-        let linecount_da = DrawingArea::new();
-        let hscrollbar = Scrollbar::new(Orientation::Horizontal, None::<&gtk::Adjustment>);
-        let vscrollbar = Scrollbar::new(Orientation::Vertical, None::<&gtk::Adjustment>);
+        let view_item = ViewItem::new();
+        let find_replace = FindReplace::new();
+        let pango_ctx = view_item.get_pango_ctx();
 
-        da.set_events(
-            EventMask::BUTTON_PRESS_MASK
-                | EventMask::BUTTON_RELEASE_MASK
-                | EventMask::BUTTON_MOTION_MASK
-                | EventMask::SCROLL_MASK
-                | EventMask::SMOOTH_SCROLL_MASK,
-        );
-        debug!("{}: {:?}", gettext("Events"), da.get_events());
-        da.set_can_focus(true);
-
-        let find_rep_src = include_str!("ui/find_replace.glade");
-        let find_rep_builder = Builder::new_from_string(find_rep_src);
-        let search_bar: SearchBar = find_rep_builder.get_object("search_bar").unwrap();
-        let replace_expander: Expander = find_rep_builder.get_object("replace_expander").unwrap();
-        let replace_revealer: Revealer = find_rep_builder.get_object("replace_revealer").unwrap();
-        let replace_entry: Entry = find_rep_builder.get_object("replace_entry").unwrap();
-        let replace_button: Button = find_rep_builder.get_object("replace_button").unwrap();
-        let replace_all_button: Button = find_rep_builder.get_object("replace_all_button").unwrap();
-        let find_status_label: Label = find_rep_builder.get_object("find_status_label").unwrap();
-
-        let replace = EVReplace {
-            replace_expander: replace_expander.clone(),
-            replace_revealer: replace_revealer.clone(),
-            replace_entry: replace_entry.clone(),
-        };
+        main_state.borrow_mut().fonts = EditView::get_font_list(&pango_ctx);
```

Comment:
Actually, I don't think we need the fonts field anymore, we don't set fonts ourselves but via the `FontChooser` gtk-rs offers.